cvat-sdk/cvat_sdk/core/proxies/annotations.py

```diff
@@ -20,7 +20,8 @@ class AnnotationCrudMixin(ABC):
     # TODO: refactor
 
     @property
-    def _put_annotations_data_param(self) -> str: ...
+    def _put_annotations_data_param(self) -> str:
```

Comment:
```suggestion
    def _put_annotations_data_param(self) -> str:
```
Probably, you used an older version of `black`.

Reply:
Yes. I updated my version, and fixed the issue